adafruit_displayio_layout/layouts/grid_layout.py

```diff
@@ -120,4 +121,24 @@ def add_content(self, cell_content, grid_position, cell_size):
             "cell_size": cell_size,
         }
         self._cell_content_list.append(sub_view_obj)
+        self._cell_dictionary[grid_position] = len(self._cell_content_list) - 1
         self._layout_cells()
+
+    def get_cell(self, cell_coordinates):
+        """
+        Return a cells content based on the cell_coordinates. Raises
+        KeyError if coordinates were not found in the GridLayout.
+
+        :param tuple cell_coordinates: the coordinates to lookup in the grid
+        :return: the displayio content object at those coordinates
+        """
```

Comment:
# create a list of grid_positions [(0,0), (0,1) ...]
    # index in cell_positions is the same as the index of the cell in self._cell_content_list.
    cell_positions = [cell['grid_position'] for cell in self._cell_content_list]
    if cell_coordinates in cell_positions:
        return self.cell_content_list[cell_positions.index(cell_coordinates)]